profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -1607,7 +1610,7 @@ function cce_basic_config_update_7192() {
   variable_set('image_captcha_distortion_amplitude', '5');
   variable_set('image_captcha_bilinear_interpolation', 1);
 
-  if (db_table_exists($captcha_points)) {
+  if (db_table_exists('captcha_points')) {
```

Comment:
It is because that block my push on CS check. $captcha_points is not instantiated.